src/hotspot/share/prims/forte.cpp

```diff
@@ -32,6 +32,7 @@
 #include "prims/jvmtiExport.hpp"
 #include "runtime/frame.inline.hpp"
 #include "runtime/javaCalls.hpp"
+#include "runtime/thread.hpp"
```

Comment:
You don't need this as you already include the thread.inline.hpp which has to include thread.hpp.